app/src/main/java/org/fossasia/openevent/app/data/EventDataRepository.java

```diff
@@ -83,6 +83,18 @@ public EventDataRepository(IUtilModel utilModel) {
         return getData(() -> eventService.getEvents(authorization), EVENTS, reload);
     }
 
+    @Override
+    public Observable<Attendee> toggleAttendeeCheckStatus(long position, long eventId, long attendeeId) {
+        return eventService.toggleAttendeeCheckStatus(eventId, attendeeId, authorization)
+            .doOnNext(attendee -> {
+                List<Attendee> attendees = (List<Attendee>) cacheModel.getValue(ATTENDEES + eventId);
```

Comment:
btw we are going to use some local database. hence for now I will just put the condition for non nullity of the cached list.